canonical/traefik-k8s-operator
# Traefik logging and grafana-dashboard integrations

This PR adds loki-logging and grafana-dashboarding integrations to the traefik charm.

## Testing Instructions
deploy cos-lite, then refresh the traefik charm with one built from these sources. Visit the grafana frontend to see the traefik dashboard with the logs.

![image](https://user-images.githubusercontent.com/6230162/211538053-ea03b5a9-dfc6-4f43-b0c2-80f197993869.png)

## Release Notes
- Added loki-logging and grafana-dashboarding integrations.
